src/chevah_compat/posix_filesystem.py

```diff
@@ -749,6 +753,10 @@ def _dirEntryToFileAttributes(self, entry):
             # On Windows, scandir gets float precision while
             # getAttributes only integer.
             modified = int(modified)
+            # On Windows, path might have long names for local drives.
+            # For compat, we keep the simple format.
+            if path.startswith('\\\\?\\') and path[5] == ':':
+                path = path[4:]
```

Comment:
If it's a known extended-length path, why remove the `\\?\` prefix?

Reply:
the high level API should just use the normal path format.
Compat will handle the low-level conversion at the low-level.

Here we are exposing a low-level path format to the high-level user.
So we normalize/clean it.

In this way, you can be sure that when you do `result = fs.iterate('c:\very-long-folder\')` the resulting member will have paths like `c:\very-long-folder\some-name.txt` and not `\\?\c:\very-long-folder\some-name.txt`

Reply:
Yes! Makes sense. This path goes towards the user side for `_dirEntryToFileAttributes`. Thank you.